docs/intro/firststeps.rst

```diff
@@ -0,0 +1,335 @@
+.. _intro-firststeps:
+
+===========
+First Steps
+===========
+
+FTPVL works best when installed in an interactive Python environment, such as a 
+`Jupyter`_ notebook. Many of the examples in the documentation will be accessible
+through notebooks hosted on `Google Colaboratory`_.
+
+You can follow along with the steps below by running the cells in 
+`this colab notebook`_.
+
+.. _intro-installing:
+
+Installing FTPVL
+================
+
+.. note::
+    It's recommended to set up a `virtual environment`_ before installing FTPVL
+    to prevent future issues with system-wide packages.
+
+Let's get started by installing FTPVL. The easiest way is to download it using 
+`PyPi`_::
+
+    pip install ftpvl
+
+
+In a Python notebook, you can also perform command-line operations by writing the
+the command into a cell prefixed with an ``!``:
+
+.. code-block:: python
+    :lineno-start: 1
+
+    !pip install ftpvl
+
+
+Now, let's import the classes that we will need to complete this tutorial:
+
+.. code-block:: python
+    :lineno-start: 2
+    
+    from ftpvl.fetchers import HydraFetcher
+    from ftpvl.processors import *
+    from ftpvl.styles import ColorMapStyle
+    from ftpvl.visualizers import SingleTableVisualizer
+
+.. _intro-fetching:
+
+Fetching Data from Hydra
+========================
+
+:ref:`topics-fetchers` in FTPVL are responsible for ingesting data from a source
+and performing simple pre-processing to standardize the output. This results in
+the creation of an :ref:`topics-evaluation` instance, which stores the fetched 
```

Comment:
That reference is to a page called `Evaluation`, so it will show up correctly once generated. See the relevant part [here](https://ftpvl.readthedocs.io/en/latest/intro/firststeps.html#fetching-data-from-hydra)